app/login/components/profile-content-tabs.tsx

```diff
@@ -66,9 +64,15 @@ export const ProfileContentTabs = (props: ProfileContentTabsProps) => {
 
   return (
     <TabContext value={value}>
-      <Box className={clsx(rootClasses.section, classes.section)}>
+      <Box className={clsx(rootClasses.section)}>
         <Box className={rootClasses.sectionContent}>
-          <Box sx={{ borderBottom: 1, borderColor: "divider" }}>
+          <Box
+            sx={{
+              borderBottom: 1,
+              borderColor: "divider",
+              marginTop: theme.spacing(6),
+            }}
+          >
```

Comment:
@noahonyejese generally I think we should prefer using `makeStyles` / `useStyles` over `sx` even if only to improve readability of JSX code by having the styling extracted. Here we already are using it, I'd bring the `section` back and even extract the `Box`'s `sx` to e.g. `tabWrapper`, so we have all styles next to each other 👍